src/encoded/static/components/static-pages/components/matrix/StackedBlockVisual.js

```diff
@@ -1374,16 +1441,49 @@ const Block = React.memo(function Block(props){
                 placement="bottom"
                 overlay={popover}
                 rootClose
```

Comment:
I pulled in the changes to test on Donor View. It looks great. Could we add a `flip={true}` here to prevent overflowing the bottom of the page?
```suggestion
                rootClose
                flip={true}
```